README.md

```diff
@@ -6,15 +6,32 @@ SourceTracker was originally described in [Knights et al., 2011](http://www.ncbi
 If you use this package, please cite the original SourceTracker paper linked
 above pending publication of SourceTracker 2.
 
+# File Formats
+
+For descriptions of all file formats and options, please see the help
+documentation, available with the command `sourcetracker2 gibbs --help`. 
+
+In brief, this script requires a feature X sample contingency table
+(traditionally an OTU table) and sample metadata (traditionally a mapping file).
+Any feature table that can be read by the `biom` package will be acceptable
+input. For example file formats please look at the test mapping file and
+feature (OTU) tables we have included at `sourcetracker2/data/tiny-test/`.
```

Comment:
You have to provide some description of what these are - pointing to QIIME makes it easy on you b/c you don't have to write much, but if you want to just describe it here that's ultimately better for users. The QIIME 2 format is going to be much more relaxed that then QIIME 1 format, so when that document exists you might just want to point to it (another motivation to just make this as easy as possible on yourself right now).